docs/framework/whats-new/obsolete-members.md

```diff
@@ -551,7 +551,7 @@ The tables in this article list the type members that are obsolete in the .NET F
 |<xref:System.Net.HttpWebRequest?displayProperty=nameWithType>|<xref:System.Net.HttpWebRequest.%23ctor>|First deprecated in the .NET Framework 4.5.<br /><br /> Use of this member generates a compiler error.<br /><br /> This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.|  
```

Comment:
line 530 - second xref link should be System.Diagnostics.PerformanceCounterPermissionAccess.Browse and second xref link should be System.Diagnostics.PerformanceCounterPermissionAccess.Read

line 531 - second xref link should be System.Diagnostics.PerformanceCounterPermissionAccess.Instrument and second xref link should be System.Diagnostics.PerformanceCounterPermissionAccess.Write